packages/cosmwasm/src/msgs.ts

```diff
@@ -144,3 +144,75 @@ export interface MsgMigrateContract extends Msg {
 export function isMsgMigrateContract(msg: Msg): msg is MsgMigrateContract {
   return (msg as MsgMigrateContract).type === "wasm/MsgMigrateContract";
 }
+
```

Comment:
All those messages 👇 live on a different level than the existing ones 👆. What we have above is sent to the chain. The new types are sent to the contract. I have no great naming suggestion that is better than the current naming, but let's put them in a different file to indicate those are different things.